rlclientlib/logger/logger_facade.cc

```diff
@@ -48,14 +69,32 @@ namespace reinforcement_learning {
       }
     }
 
+    int ccb_logger_facade::log_decisions(const char* event_id, const char* context, unsigned int flags, const std::vector<std::vector<uint32_t>>& action_ids,
+      const std::vector<std::vector<float>>& pdfs, const std::string& model_version, api_status* status) {
+      switch (version) {
+        case 2: return v2->log(event_id, serializer.event(context, flags, action_ids, pdfs, model_version), serializer.type, status);
```

Comment:
Why no v1 here? Shouldn't we fail differently in that case?

Reply:
As we discussed, there is a breaking change in CCB api (from unique id per slot to event id + slot index).
Because of that,
CCB methods which are using event_id + slot index as key support only v2 protocol, and 
CCB methods which are using unique slot ids support only v1 protocol.
v2 CCB methods are not exposed to client yet.

Reply:
Makes sense.
We need to mark as obsolete all v1 methods in the API(s) then.

Reply:
agree (but when v2 are exposed on API level - not in this PR)

Reply:
will it be under a flag initially?

Reply:
not 100% sure what do you mean - can you please elaborate?